proposals/compatibility-mode.md

```diff
@@ -89,11 +89,17 @@ A bind group may not reference a subset of array layers. Only views of the entir
 
 **Justification**: OpenGL ES does not support texture views.
 
-### 7. Disallow `sample_mask` builtin in WGSL.
+### 7. Disallow `sample_mask` and `sample_index` builtins in WGSL.
 
-Use of the `sample_mask` builtin would cause a validation error at pipeline creation time.
+Use of the `sample_mask` or `sample_index` builtins would cause a validation error at pipeline creation time.
 
-**Justification**: OpenGL ES 3.1 does not support `gl_SampleMask` or `gl_SampleMaskIn`.
+**Justification**: OpenGL ES 3.1 does not support `gl_SampleMask`, `gl_SampleMaskIn`, or `gl_SampleID`.
+
+### 8. Disallow two-component (RG) texture formats in storage texture bindings.
```

Comment:
The suggestion in https://github.com/gpuweb/gpuweb/issues/4413 is that these texture formats be marked as not supporting storage in in compat (see: the texture-format-capabilities table [here](https://www.w3.org/TR/webgpu/#texture-format-caps). 

This means `createTexture`, `createBindGroupLayout`, `createRenderPipeline`, `createRenderPipelineAsync`, `createComputePipeline`, and `createComputePipelineAsync` all generate validation errors for these textures if they are used a storage textures.

Reply:
Thanks! Fixed.